foyer/general_forcefield.py

```diff
@@ -34,17 +36,18 @@ def preprocess_forcefield_files(forcefield_files=None, backend="gmso"):
             _, suffix = os.path.split(file)
             tempfile = NamedTemporaryFile(suffix=suffix, delete=False)
             try:
+                Validator(ff_file_name=file, debug=False)
                 from_foyer_xml(
                     foyer_xml=str(file),
                     gmso_xml=str(tempfile.name),
                     overwrite=True,
                 )
-                tmp_processed_files.append(tempfile.name)
-            except Exception:
+            except:
```

Comment:
Should this be a ValidationError? That's what it seems like the Validator is setup to raise, see [line](https://github.com/mosdef-hub/foyer/blob/c5802ba3dca63c310a309c6b74e5705e4f7ae83b/foyer/validator.py#L74) in validator.py. Might be better to not raise such a general exception, unless there are other errors that might arise from the validation.

Reply:
The current foyer validator is more to check if an atom type is provided provide a malformed SMARTS, or is missing a nonbonded, etc. Trying to validating a GMSO XML will raise error, but not ValidationError, more like XML parsing error (`This element is not expected`). So I think having a general catch here is ok, since if there is any other issues, it would be raised by the GMSO validator.